### ISSUE TITLE
Gets stuck on "Parsing: Statistics" with following stat block due to semi-colon

### ISSUE THREAD

**Issue by BillGr7** [Lavaeolous/PF1-StatBlock-Converter-Module#488]
When I paste in the following stat block copied from The Ruby Phoenix Tournament module, sbc seems to get stuck at "Parsing: Statistics". The stat block is:
```
Orometsu Yasi CR 8
XP 4,800
Female human fighter 9
LN Medium humanoid (human)
Init +7; Senses Perception +1
DEFENSE
AC 22, touch 14, flat-footed 18 (+8 armor, +3 Dex, +1 dodge)
hp 90 (9d10+36)
Fort +8, Ref +6, Will +4; +2 vs. fear
Defensive Abilities bravery +2
OFFENSE
Speed 30 ft.
Melee +1 frost nodachi +17/+12 (1d10+9/15–20 plus 1d6 cold)
Ranged mwk shortbow +14/+9 (1d6+1/×3)
Special Attacks weapon training (heavy blades +2, bows +1)
STATISTICS
Str 16, Dex 16, Con 14, Int 10, Wis 12, Cha 8
Base Atk +9; CMB +12; CMD 26
Feats Combat Reflexes, Dodge, Greater Weapon Focus(nodachi), Improved Critical (nodachi), Improved Initiative, Mobility, Power Attack, Spring Attack, Toughness, Weapon Focus (nodachi), Weapon Specialization (nodachi)
Skills Climb +6, Diplomacy +8, Intimidate +7, Knowledge (nobility) +4, Ride +7
Languages Minkaian, Tien
SQ armor training 2
Combat Gear potion of cure moderate wounds, potion of haste;
Other Gear +1 kusari gusoku, +1 frost nodachi, slippers of spider climbing, masterwork shortbow with 20 arrows
```

I did some testing, and it seems like its specifically the semi-colon at the end of the "Combat Gear" line that's causing it to hang.  If I delete that, it doesn't get stuck, but then doesn't add the items from the "Other Gear" section. I ended up combining the two gear sections as a workaround.

**Comment by Lavaeolous** [comment]
Still valid, review.

**Comment by Lavaeolous** [comment]
Fixed in 3.3.4
